WISE-Community/WISE
# Temporarily hid links to missing features on the new site

Also fixed a few visual, text, and layout bugs.

To test:
- Load the site and make sure all links to the About Us page have been removed.
- Sign in as a teacher and open the run settings menu for an active, completed, and future run.
- Make sure the 'End Unit', 'Restart Unit', and 'Cancel Unit' menu items are hidden.

Closes #1647.
